packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -85,6 +85,8 @@ class CreateCommand extends FlutterCommand {
         getEnumName(_ProjectType.plugin): 'Generate a shareable Flutter project containing an API '
             'in Dart code with a platform-specific implementation for Android, for iOS code, or '
             'for both.',
+        getEnumName(_ProjectType.module): 'Generate a Flutter project to add Flutter views to a non-'
```

Comment:
I couldn't find an official sentence for this, so I made it up.  Open to suggestions, @xster.

Reply:
How about `Generate a project to add a Flutter module to existing Android or iOS applications`?

@RedBrogdon?

Reply:
@zanderso

Reply:
@zanderso

Reply:
I like @xster's suggestion with a tiny change:
`Generate a project to add a Flutter module to an existing Android or iOS application.`

Reply:
I like that, thanks!